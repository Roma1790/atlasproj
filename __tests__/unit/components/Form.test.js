import React from "react"
import renderer from "react-test-renderer"

import Form, { createSelectOptions } from "../../../src/components/Form"

describe("Form", () => {
  describe("Snapshot", () => {
    it("should render correctly", () => {
      const component = renderer.create(<Form />)
      expect(component.toJSON()).toMatchSnapshot()
    })
  })

  describe("createSelectOptions()", () => {
    it("should return the correct options for more than 1 option", () => {
      const testCases = [
        {
          in: ["AAAA", "BBBB", "CCCC", "DDDD", "EEEE", "FFFF"],
          snapshot: `
          Array [
            <option
              value="AAAA"
            >
              AAAA
            </option>,
            <option
              value="BBBB"
            >
              BBBB
            </option>,
            <option
              value="CCCC"
            >
              CCCC
            </option>,
            <option
              value="DDDD"
            >
              DDDD
            </option>,
            <option
              value="EEEE"
            >
              EEEE
            </option>,
            <option
              value="FFFF"
            >
              FFFF
            </option>,
          ]
          `,
        },
        {
          in: ["Angewandte Mathematik, Physik und Allgemeinwissenschaften", "Architektur", "Bauingenieurwesen"],
          snapshot: `
          Array [
            <option
              value="Angewandte Mathematik, Physik und Allgemeinwissenschaften"
            >
              Angewandte Mathematik, Physik und Allgemeinwissenschaften
            </option>,
            <option
              value="Architektur"
            >
              Architektur
            </option>,
            <option
              value="Bauingenieurwesen"
            >
              Bauingenieurwesen
            </option>,
          ]
          `,
        },
      ]
      testCases.forEach(tc => {
        expect(createSelectOptions(tc.in)).toMatchInlineSnapshot(tc.snapshot)
      })
    })

    it("should return the correct options for 1 option", () => {
      const testCase = {
        in: ["AAAA"],
        snapshot: `
          Array [
            <option
              value="AAAA"
            >
              AAAA
            </option>,
          ]
          `,
      }
      expect(createSelectOptions(testCase.in)).toMatchInlineSnapshot(testCase.snapshot)
    })

    it("should return an empty string if no options are provided", () => {
      expect(createSelectOptions([])).toEqual([])
    })
  })
})
