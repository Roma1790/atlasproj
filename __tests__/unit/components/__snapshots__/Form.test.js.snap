// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Snapshot should render correctly 1`] = `
<form>
  <div
    className="flex flex-wrap items-stretch w-full mb-4 relative"
  >
    <div
      className="flex -mr-px"
    >
      <span
        className="flex items-center leading-normal bg-grey-lighter rounded rounded-r-none border border-r-0 border-grey-light px-3 whitespace-no-wrap text-grey-dark text-sm"
      >
        Praktikumsart:
      </span>
    </div>
    <input
      className="flex-shrink flex-grow flex-auto leading-normal w-px flex-1 border h-10 border-grey-light rounded rounded-l-none px-3 relative focus:border-blue focus:shadow"
      placeholder="Username"
      type="text"
    />
  </div>
  <div
    className="bg-white shadow-md rounded px-8 pt-6 pb-8 mb-4 flex flex-col my-2"
  >
    <div
      className="-mx-3 md:flex mb-6"
    >
      <div
        className="md:w-1/2 px-3 mb-6 md:mb-0"
      >
        <label
          className="block uppercase tracking-wide text-grey-darker text-xs font-bold mb-2"
        >
          First Name
        </label>
        <span>
          @
        </span>
        <input
          className="appearance-none block w-full bg-grey-lighter text-grey-darker border border-red rounded py-3 px-4 mb-3"
          id="grid-first-name"
          placeholder="Jane"
          type="text"
        />
        <p
          className="text-red text-xs italic"
        >
          Please fill out this field.
        </p>
      </div>
      <div
        className="md:w-1/2 px-3"
      >
        <label
          className="block uppercase tracking-wide text-grey-darker text-xs font-bold mb-2"
        >
          Last Name
        </label>
        <input
          className="appearance-none block w-full bg-grey-lighter text-grey-darker border border-grey-lighter rounded py-3 px-4"
          id="grid-last-name"
          placeholder="Doe"
          type="text"
        />
      </div>
    </div>
    <div
      className="-mx-3 md:flex mb-6"
    >
      <div
        className="md:w-full px-3"
      >
        <label
          className="block uppercase tracking-wide text-grey-darker text-xs font-bold mb-2"
        >
          Password
        </label>
        <input
          className="appearance-none block w-full bg-grey-lighter text-grey-darker border border-grey-lighter rounded py-3 px-4 mb-3"
          id="grid-password"
          placeholder="******************"
          type="password"
        />
        <p
          className="text-grey-dark text-xs italic"
        >
          Make it as long and as crazy as you'd like
        </p>
      </div>
    </div>
    <div
      className="-mx-3 md:flex mb-2"
    >
      <div
        className="md:w-1/2 px-3"
      >
        <label
          className="block uppercase tracking-wide text-grey-darker text-xs font-bold mb-2"
        >
          State
        </label>
        <div
          className="relative"
        >
          <select
            className="block appearance-none w-full bg-grey-lighter border border-grey-lighter text-grey-darker py-3 px-4 pr-8 rounded"
            id="grid-state"
          >
            <option>
              New Mexico
            </option>
            <option>
              Missouri
            </option>
            <option>
              Texas
            </option>
          </select>
          <div
            className="pointer-events-none absolute pin-y pin-r flex items-center px-2 text-grey-darker"
          >
            <svg
              className="h-4 w-4"
              viewBox="0 0 20 20"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex flex-wrap -mx-3 mb-6"
  >
    <div
      className="w-full md:w-1/2 px-3 mb-6 md:mb-0"
    >
      <label
        className="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
      >
        Was?
      </label>
      <label
        className="text-gray-700 text-xs font-bold mb-2"
      >
        Praktikumsart
      </label>
      <select
        className="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
        id="grid-state"
      >
        <option
          value="Praktikum im Studium"
        >
          Praktikum im Studium
        </option>
        <option
          value="Bachelor- oder Masterarbeit"
        >
          Bachelor- oder Masterarbeit
        </option>
        <option
          value="Stellen f&uuml;r Doktoranden und Doktorandinnen"
        >
          Stellen f&uuml;r Doktoranden und Doktorandinnen
        </option>
        <option
          value="Stellen f&uuml;r Duales Studium"
        >
          Stellen f&uuml;r Duales Studium
        </option>
        <option
          value="Werkstudentent&auml;tigkeit"
        >
          Werkstudentent&auml;tigkeit
        </option>
        <option
          value="Studentenjob"
        >
          Studentenjob
        </option>
        <option
          value="Stelle f&uuml;r Absolventen und Absolventinnen"
        >
          Stelle f&uuml;r Absolventen und Absolventinnen
        </option>
        <option
          value="Traineeprogramm"
        >
          Traineeprogramm
        </option>
        <option
          value="Stelle, die erste Berufserfahrung voraussetzt"
        >
          Stelle, die erste Berufserfahrung voraussetzt
        </option>
        <option
          value="Stellen f&uuml;r Studienabbrecher"
        >
          Stellen f&uuml;r Studienabbrecher
        </option>
      </select>
      <select
        className="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
        id="grid-state"
      >
        <option
          value="Angewandte Chemie"
        >
          Angewandte Chemie
        </option>
        <option
          value="Angewandte Mathematik, Physik und Allgemeinwissenschaften"
        >
          Angewandte Mathematik, Physik und Allgemeinwissenschaften
        </option>
        <option
          value="Architektur"
        >
          Architektur
        </option>
        <option
          value="Bauingenieurwesen"
        >
          Bauingenieurwesen
        </option>
        <option
          value="Betriebswirtschaft"
        >
          Betriebswirtschaft
        </option>
        <option
          value="Design"
        >
          Design
        </option>
        <option
          value="Elektrotechnik Feinwerktechnik Informationstechnik"
        >
          Elektrotechnik Feinwerktechnik Informationstechnik
        </option>
        <option
          value="Informatik"
        >
          Informatik
        </option>
        <option
          value="Maschinenbau und Versorgungstechnik"
        >
          Maschinenbau und Versorgungstechnik
        </option>
        <option
          value="Sozialwissenschaften"
        >
          Sozialwissenschaften
        </option>
        <option
          value="Verfahrenstechnik"
        >
          Verfahrenstechnik
        </option>
        <option
          value="Werkstofftechnik"
        >
          Werkstofftechnik
        </option>
      </select>
      <select
        className="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
        id="grid-state"
      >
        <option
          value="Abfallwirtschaft"
        >
          Abfallwirtschaft
        </option>
        <option
          value="Anlagen- und Maschinenbau"
        >
          Anlagen- und Maschinenbau
        </option>
        <option
          value="Automatisierungstechnik"
        >
          Automatisierungstechnik
        </option>
        <option
          value="Automobilbau"
        >
          Automobilbau
        </option>
        <option
          value="Banken, Versicherung, Immobilien,"
        >
          Banken, Versicherung, Immobilien,
        </option>
        <option
          value="Finanzdienstleistung"
        >
          Finanzdienstleistung
        </option>
        <option
          value="Bauwesen, Architektur"
        >
          Bauwesen, Architektur
        </option>
        <option
          value="Bergbau"
        >
          Bergbau
        </option>
        <option
          value="Bildung&amp;Training"
        >
          Bildung&amp;Training
        </option>
        <option
          value="Biotechnik, Pharmazie"
        >
          Biotechnik, Pharmazie
        </option>
        <option
          value="Chemie- und erd&ouml;lverarbeitende Industrie"
        >
          Chemie- und erd&ouml;lverarbeitende Industrie
        </option>
        <option
          value="Druck, Papier und Verpackungsindustrie"
        >
          Druck, Papier und Verpackungsindustrie
        </option>
        <option
          value="Einzel-, Gro&szlig;- und Au&szlig;enhandel"
        >
          Einzel-, Gro&szlig;- und Au&szlig;enhandel
        </option>
        <option
          value="Elektrotechnik, Feinmechanik, Optik"
        >
          Elektrotechnik, Feinmechanik, Optik
        </option>
        <option
          value="Energie-/Wasserversorgung"
        >
          Energie-/Wasserversorgung
        </option>
        <option
          value="Fahrzeugbau/-zulieferer"
        >
          Fahrzeugbau/-zulieferer
        </option>
        <option
          value="Gastronomie, Touristik"
        >
          Gastronomie, Touristik
        </option>
        <option
          value="Gesundheit, Soziale Dienste"
        >
          Gesundheit, Soziale Dienste
        </option>
        <option
          value="Glas, Keramik"
        >
          Glas, Keramik
        </option>
        <option
          value="Holz- und M&ouml;belindustrie"
        >
          Holz- und M&ouml;belindustrie
        </option>
        <option
          value="Ingenieurdienstleistung"
        >
          Ingenieurdienstleistung
        </option>
        <option
          value="Internet"
        >
          Internet
        </option>
        <option
          value="IT-Dienstleistung, EDV-Schulung"
        >
          IT-Dienstleistung, EDV-Schulung
        </option>
        <option
          value="IT-Hardware – Programmierung und Design"
        >
          IT-Hardware – Programmierung und Design
        </option>
        <option
          value="IT-Softwareentwicklung"
        >
          IT-Softwareentwicklung
        </option>
        <option
          value="Konsumg&uuml;ter"
        >
          Konsumg&uuml;ter
        </option>
        <option
          value="Kunst, Kultur"
        >
          Kunst, Kultur
        </option>
        <option
          value="Kunststoffindustrie, -verarbeitung"
        >
          Kunststoffindustrie, -verarbeitung
        </option>
        <option
          value="Land-, Forst-, Fischwirtschaft"
        >
          Land-, Forst-, Fischwirtschaft
        </option>
        <option
          value="Luft- und Raumfahrt"
        >
          Luft- und Raumfahrt
        </option>
        <option
          value="Management, Beratung"
        >
          Management, Beratung
        </option>
        <option
          value="Marketing, Werbung &amp; PR"
        >
          Marketing, Werbung &amp; PR
        </option>
        <option
          value="Marktforschung"
        >
          Marktforschung
        </option>
        <option
          value="Medien"
        >
          Medien
        </option>
        <option
          value="Medizintechnik"
        >
          Medizintechnik
        </option>
        <option
          value="Metall"
        >
          Metall
        </option>
        <option
          value="Nahrungs- und Genussmittel"
        >
          Nahrungs- und Genussmittel
        </option>
        <option
          value="&Ouml;ffentlicher Dienst und Verb&auml;nde"
        >
          &Ouml;ffentlicher Dienst und Verb&auml;nde
        </option>
        <option
          value="Personaldienstleistung"
        >
          Personaldienstleistung
        </option>
        <option
          value="Petrochemie"
        >
          Petrochemie
        </option>
        <option
          value="Recht, Steuern, Finanzen, Controlling"
        >
          Recht, Steuern, Finanzen, Controlling
        </option>
        <option
          value="Sonstige Branche"
        >
          Sonstige Branche
        </option>
        <option
          value="Sonstige Dienstleistung"
        >
          Sonstige Dienstleistung
        </option>
        <option
          value="Telekommunikation"
        >
          Telekommunikation
        </option>
        <option
          value="Textil, Leder, Bekleidung"
        >
          Textil, Leder, Bekleidung
        </option>
        <option
          value="Verkehr, Lager, Logistik"
        >
          Verkehr, Lager, Logistik
        </option>
        <option
          value="Wissenschaft, Forschung &amp; Entwicklung"
        >
          Wissenschaft, Forschung &amp; Entwicklung
        </option>
      </select>
      <input
        className="w-full"
        placeholder="Stichworte"
        type="text"
      />
    </div>
    <div
      className="w-full md:w-1/2 px-3 mb-6 md:mb-0"
    >
      <label
        className="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
      >
        Wo?
      </label>
      <input
        className="appearance-none bg-transparent border-none w-full text-gray-700 mr-3 py-1 px-2 leading-tight focus:outline-none"
        placeholder="Ort oder Adresse"
        type="text"
      />
      Umkreissuche mit 
      <input
        type="number"
        value="10"
      />
      km Radius.
    </div>
  </div>
  <label
    className="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
  >
    Und Ausserdem...
  </label>
  <div
    className="flex flex-wrap"
  >
    <select
      className="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
      id="grid-state"
      placeholder="Vertragsart"
    >
      <option
        value="Befristet"
      >
        Befristet
      </option>
      <option
        value="Festanstellung"
      >
        Festanstellung
      </option>
      <option
        value="Freie Mitarbeit"
      >
        Freie Mitarbeit
      </option>
      <option
        value="Projektarbeit"
      >
        Projektarbeit
      </option>
      <option
        value="Teilzeit"
      >
        Teilzeit
      </option>
      <option
        value="Vollzeit"
      >
        Vollzeit
      </option>
    </select>
    <select
      className="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
      id="grid-state"
    >
      <option
        value="Deutsch"
      >
        Deutsch
      </option>
      <option
        value="Deutscher Dialekt (Schweizerdeutsch)"
      >
        Deutscher Dialekt (Schweizerdeutsch)
      </option>
      <option
        value="Englisch"
      >
        Englisch
      </option>
      <option
        value="Englisch (Technisches Englisch)"
      >
        Englisch (Technisches Englisch)
      </option>
      <option
        value="Englisch (Wirtschafts- oder Businessenglisch)d"
      >
        Englisch (Wirtschafts- oder Businessenglisch)d
      </option>
      <option
        value="Spanisch"
      >
        Spanisch
      </option>
      <option
        value="Franz&ouml;sisch"
      >
        Franz&ouml;sisch
      </option>
      <option
        value="Afrikaansd"
      >
        Afrikaansd
      </option>
      <option
        value="Akan/Twi (Westafrika)d"
      >
        Akan/Twi (Westafrika)d
      </option>
      <option
        value="Albanischd"
      >
        Albanischd
      </option>
      <option
        value="Amharisch (&Auml;thiopien)"
      >
        Amharisch (&Auml;thiopien)
      </option>
      <option
        value="Arabischd"
      >
        Arabischd
      </option>
      <option
        value="Aram&auml;isch/Syrischd"
      >
        Aram&auml;isch/Syrischd
      </option>
      <option
        value="Armenischd"
      >
        Armenischd
      </option>
      <option
        value="Aserbaidschanischd"
      >
        Aserbaidschanischd
      </option>
      <option
        value="Bambara (Westafrika, Mali)"
      >
        Bambara (Westafrika, Mali)
      </option>
      <option
        value="Bengalid"
      >
        Bengalid
      </option>
      <option
        value="Birmanischd"
      >
        Birmanischd
      </option>
      <option
        value="Bosnischd"
      >
        Bosnischd
      </option>
      <option
        value="Bulgarischd"
      >
        Bulgarischd
      </option>
      <option
        value="Chinesischd"
      >
        Chinesischd
      </option>
      <option
        value="D&auml;nischd"
      >
        D&auml;nischd
      </option>
      <option
        value="Estnischd"
      >
        Estnischd
      </option>
      <option
        value="Filipino (Tagalog)d"
      >
        Filipino (Tagalog)d
      </option>
      <option
        value="Finnischd"
      >
        Finnischd
      </option>
      <option
        value="Georgischd"
      >
        Georgischd
      </option>
      <option
        value="Griechischd"
      >
        Griechischd
      </option>
      <option
        value="Gujaratid"
      >
        Gujaratid
      </option>
      <option
        value="Hausa (Westafrika)"
      >
        Hausa (Westafrika)
      </option>
      <option
        value="Hebr&auml;ischd"
      >
        Hebr&auml;ischd
      </option>
      <option
        value="Hindid"
      >
        Hindid
      </option>
      <option
        value="Indonesischd"
      >
        Indonesischd
      </option>
      <option
        value="Isl&auml;ndischd"
      >
        Isl&auml;ndischd
      </option>
      <option
        value="Italienisch"
      >
        Italienisch
      </option>
      <option
        value="Jamaika-Kreolisch (Patois)"
      >
        Jamaika-Kreolisch (Patois)
      </option>
      <option
        value="Kambodschanisch (Khmer)d"
      >
        Kambodschanisch (Khmer)d
      </option>
      <option
        value="Kasachischd"
      >
        Kasachischd
      </option>
      <option
        value="Kirgisischd"
      >
        Kirgisischd
      </option>
      <option
        value="Koreanischd"
      >
        Koreanischd
      </option>
      <option
        value="Kroatischd"
      >
        Kroatischd
      </option>
      <option
        value="Kurdischd"
      >
        Kurdischd
      </option>
      <option
        value="Laotischd"
      >
        Laotischd
      </option>
      <option
        value="Lettischd"
      >
        Lettischd
      </option>
      <option
        value="Letzeburgisch (Luxemburg)"
      >
        Letzeburgisch (Luxemburg)
      </option>
      <option
        value="Malayalamd"
      >
        Malayalamd
      </option>
      <option
        value="Marathid"
      >
        Marathid
      </option>
      <option
        value="Mazedonischd"
      >
        Mazedonischd
      </option>
      <option
        value="Mongolischd"
      >
        Mongolischd
      </option>
      <option
        value="Nepalesischd"
      >
        Nepalesischd
      </option>
      <option
        value="Niederl&auml;ndisch/Fl&auml;misch"
      >
        Niederl&auml;ndisch/Fl&auml;misch
      </option>
      <option
        value="Pakistanisch/Urdud"
      >
        Pakistanisch/Urdud
      </option>
      <option
        value="Paschtu/Paschto/Pashto (Afghanistan)"
      >
        Paschtu/Paschto/Pashto (Afghanistan)
      </option>
      <option
        value="Portugiesischd"
      >
        Portugiesischd
      </option>
      <option
        value="Punjabid"
      >
        Punjabid
      </option>
      <option
        value="Romanid"
      >
        Romanid
      </option>
      <option
        value="Rum&auml;nischd"
      >
        Rum&auml;nischd
      </option>
      <option
        value="Russisch"
      >
        Russisch
      </option>
      <option
        value="Schwedischd"
      >
        Schwedischd
      </option>
      <option
        value="Serbischd"
      >
        Serbischd
      </option>
      <option
        value="Singhalesischd"
      >
        Singhalesischd
      </option>
      <option
        value="Slowakischd"
      >
        Slowakischd
      </option>
      <option
        value="Slowenischd"
      >
        Slowenischd
      </option>
      <option
        value="Somalid"
      >
        Somalid
      </option>
      <option
        value="Sorbisch/Wendischd"
      >
        Sorbisch/Wendischd
      </option>
      <option
        value="Suaheli (Ostafrika)d"
      >
        Suaheli (Ostafrika)d
      </option>
      <option
        value="Taiwanischd"
      >
        Taiwanischd
      </option>
      <option
        value="Tamild"
      >
        Tamild
      </option>
      <option
        value="Telugud"
      >
        Telugud
      </option>
      <option
        value="Thaid"
      >
        Thaid
      </option>
      <option
        value="Tibetischd"
      >
        Tibetischd
      </option>
      <option
        value="Tigrinja (Eritrea)d"
      >
        Tigrinja (Eritrea)d
      </option>
      <option
        value="Tschechischd"
      >
        Tschechischd
      </option>
      <option
        value="Tschetschenischd"
      >
        Tschetschenischd
      </option>
      <option
        value="T&uuml;rkisch"
      >
        T&uuml;rkisch
      </option>
      <option
        value="Uigurischd"
      >
        Uigurischd
      </option>
      <option
        value="Ukrainischd"
      >
        Ukrainischd
      </option>
      <option
        value="Ungarischd"
      >
        Ungarischd
      </option>
      <option
        value="Usbekischd"
      >
        Usbekischd
      </option>
      <option
        value="Vietnamesischd"
      >
        Vietnamesischd
      </option>
      <option
        value="Wei&szlig;russisch"
      >
        Wei&szlig;russisch
      </option>
      <option
        value="Yoruba (Westafrika)"
      >
        Yoruba (Westafrika)
      </option>
    </select>
    <select
      className="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
      id="grid-state"
    >
      <option
        value="Deutsch"
      >
        Deutsch
      </option>
      <option
        value="Deutscher Dialekt (Schweizerdeutsch)"
      >
        Deutscher Dialekt (Schweizerdeutsch)
      </option>
      <option
        value="Englisch"
      >
        Englisch
      </option>
      <option
        value="Englisch (Technisches Englisch)"
      >
        Englisch (Technisches Englisch)
      </option>
      <option
        value="Englisch (Wirtschafts- oder Businessenglisch)d"
      >
        Englisch (Wirtschafts- oder Businessenglisch)d
      </option>
      <option
        value="Spanisch"
      >
        Spanisch
      </option>
      <option
        value="Franz&ouml;sisch"
      >
        Franz&ouml;sisch
      </option>
      <option
        value="Afrikaansd"
      >
        Afrikaansd
      </option>
      <option
        value="Akan/Twi (Westafrika)d"
      >
        Akan/Twi (Westafrika)d
      </option>
      <option
        value="Albanischd"
      >
        Albanischd
      </option>
      <option
        value="Amharisch (&Auml;thiopien)"
      >
        Amharisch (&Auml;thiopien)
      </option>
      <option
        value="Arabischd"
      >
        Arabischd
      </option>
      <option
        value="Aram&auml;isch/Syrischd"
      >
        Aram&auml;isch/Syrischd
      </option>
      <option
        value="Armenischd"
      >
        Armenischd
      </option>
      <option
        value="Aserbaidschanischd"
      >
        Aserbaidschanischd
      </option>
      <option
        value="Bambara (Westafrika, Mali)"
      >
        Bambara (Westafrika, Mali)
      </option>
      <option
        value="Bengalid"
      >
        Bengalid
      </option>
      <option
        value="Birmanischd"
      >
        Birmanischd
      </option>
      <option
        value="Bosnischd"
      >
        Bosnischd
      </option>
      <option
        value="Bulgarischd"
      >
        Bulgarischd
      </option>
      <option
        value="Chinesischd"
      >
        Chinesischd
      </option>
      <option
        value="D&auml;nischd"
      >
        D&auml;nischd
      </option>
      <option
        value="Estnischd"
      >
        Estnischd
      </option>
      <option
        value="Filipino (Tagalog)d"
      >
        Filipino (Tagalog)d
      </option>
      <option
        value="Finnischd"
      >
        Finnischd
      </option>
      <option
        value="Georgischd"
      >
        Georgischd
      </option>
      <option
        value="Griechischd"
      >
        Griechischd
      </option>
      <option
        value="Gujaratid"
      >
        Gujaratid
      </option>
      <option
        value="Hausa (Westafrika)"
      >
        Hausa (Westafrika)
      </option>
      <option
        value="Hebr&auml;ischd"
      >
        Hebr&auml;ischd
      </option>
      <option
        value="Hindid"
      >
        Hindid
      </option>
      <option
        value="Indonesischd"
      >
        Indonesischd
      </option>
      <option
        value="Isl&auml;ndischd"
      >
        Isl&auml;ndischd
      </option>
      <option
        value="Italienisch"
      >
        Italienisch
      </option>
      <option
        value="Jamaika-Kreolisch (Patois)"
      >
        Jamaika-Kreolisch (Patois)
      </option>
      <option
        value="Kambodschanisch (Khmer)d"
      >
        Kambodschanisch (Khmer)d
      </option>
      <option
        value="Kasachischd"
      >
        Kasachischd
      </option>
      <option
        value="Kirgisischd"
      >
        Kirgisischd
      </option>
      <option
        value="Koreanischd"
      >
        Koreanischd
      </option>
      <option
        value="Kroatischd"
      >
        Kroatischd
      </option>
      <option
        value="Kurdischd"
      >
        Kurdischd
      </option>
      <option
        value="Laotischd"
      >
        Laotischd
      </option>
      <option
        value="Lettischd"
      >
        Lettischd
      </option>
      <option
        value="Letzeburgisch (Luxemburg)"
      >
        Letzeburgisch (Luxemburg)
      </option>
      <option
        value="Malayalamd"
      >
        Malayalamd
      </option>
      <option
        value="Marathid"
      >
        Marathid
      </option>
      <option
        value="Mazedonischd"
      >
        Mazedonischd
      </option>
      <option
        value="Mongolischd"
      >
        Mongolischd
      </option>
      <option
        value="Nepalesischd"
      >
        Nepalesischd
      </option>
      <option
        value="Niederl&auml;ndisch/Fl&auml;misch"
      >
        Niederl&auml;ndisch/Fl&auml;misch
      </option>
      <option
        value="Pakistanisch/Urdud"
      >
        Pakistanisch/Urdud
      </option>
      <option
        value="Paschtu/Paschto/Pashto (Afghanistan)"
      >
        Paschtu/Paschto/Pashto (Afghanistan)
      </option>
      <option
        value="Portugiesischd"
      >
        Portugiesischd
      </option>
      <option
        value="Punjabid"
      >
        Punjabid
      </option>
      <option
        value="Romanid"
      >
        Romanid
      </option>
      <option
        value="Rum&auml;nischd"
      >
        Rum&auml;nischd
      </option>
      <option
        value="Russisch"
      >
        Russisch
      </option>
      <option
        value="Schwedischd"
      >
        Schwedischd
      </option>
      <option
        value="Serbischd"
      >
        Serbischd
      </option>
      <option
        value="Singhalesischd"
      >
        Singhalesischd
      </option>
      <option
        value="Slowakischd"
      >
        Slowakischd
      </option>
      <option
        value="Slowenischd"
      >
        Slowenischd
      </option>
      <option
        value="Somalid"
      >
        Somalid
      </option>
      <option
        value="Sorbisch/Wendischd"
      >
        Sorbisch/Wendischd
      </option>
      <option
        value="Suaheli (Ostafrika)d"
      >
        Suaheli (Ostafrika)d
      </option>
      <option
        value="Taiwanischd"
      >
        Taiwanischd
      </option>
      <option
        value="Tamild"
      >
        Tamild
      </option>
      <option
        value="Telugud"
      >
        Telugud
      </option>
      <option
        value="Thaid"
      >
        Thaid
      </option>
      <option
        value="Tibetischd"
      >
        Tibetischd
      </option>
      <option
        value="Tigrinja (Eritrea)d"
      >
        Tigrinja (Eritrea)d
      </option>
      <option
        value="Tschechischd"
      >
        Tschechischd
      </option>
      <option
        value="Tschetschenischd"
      >
        Tschetschenischd
      </option>
      <option
        value="T&uuml;rkisch"
      >
        T&uuml;rkisch
      </option>
      <option
        value="Uigurischd"
      >
        Uigurischd
      </option>
      <option
        value="Ukrainischd"
      >
        Ukrainischd
      </option>
      <option
        value="Ungarischd"
      >
        Ungarischd
      </option>
      <option
        value="Usbekischd"
      >
        Usbekischd
      </option>
      <option
        value="Vietnamesischd"
      >
        Vietnamesischd
      </option>
      <option
        value="Wei&szlig;russisch"
      >
        Wei&szlig;russisch
      </option>
      <option
        value="Yoruba (Westafrika)"
      >
        Yoruba (Westafrika)
      </option>
    </select>
    <input
      className="appearance-none bg-transparent border-none w-full text-gray-700 mr-3 py-1 px-2 leading-tight focus:outline-none"
      placeholder="Ohne diese Stichworte"
      type="text"
    />
  </div>
</form>
`;
